liveclasses/views.py

```
@@ -0,0 +1,68 @@
+from django.shortcuts import render, get_object_or_404, redirect
+from django.contrib.auth.decorators import login_required
+from django.contrib import messages
+from django.utils.timezone import now
```

Ruff (0.8.2)

`django.utils.timezone.now` imported but unused

Remove unused import: `django.utils.timezone.now`

(F401)